stake-o-matic/src/main.rs

```diff
@@ -350,34 +359,15 @@ fn classify_block_producers(
 
     let leader_schedule = rpc_client.get_leader_schedule(Some(first_slot))?.unwrap();
 
-    let mut confirmed_blocks = vec![];
-    // Fetching a large number of blocks from BigTable can cause timeouts, break up the requests
-    const LONGTERM_STORAGE_STEP: u64 = 5_000;
-    let mut next_slot = first_slot;
-    while next_slot < last_slot_in_epoch {
-        let last_slot = if next_slot >= minimum_ledger_slot {
-            last_slot_in_epoch
-        } else {
-            last_slot_in_epoch.min(next_slot + LONGTERM_STORAGE_STEP)
-        };
-        let slots_remaining = last_slot_in_epoch - last_slot;
-        info!(
-            "Fetching confirmed blocks between {} - {}{}",
-            next_slot,
-            last_slot,
-            if slots_remaining > 0 {
-                format!(" ({} remaining)", slots_remaining)
-            } else {
-                "".to_string()
-            }
-        );
-
-        confirmed_blocks.push(retry_rpc_operation(42, || {
-            rpc_client.get_confirmed_blocks(next_slot, Some(last_slot))
-        })?);
-        next_slot = last_slot + 1;
-    }
-    let confirmed_blocks: HashSet<Slot> = confirmed_blocks.into_iter().flatten().collect();
+    let home_dir = std::env::var("HOME").unwrap();
+    let cache_path = PathBuf::from(home_dir)
```

Comment:
Can you sneak in a command-line arg to specify the cache location, and then use this path as the default.  This'll (1) give us a knob, (2) documents the default path in `--help` for easy discovery later